lib/jekyll-seo-tag/drop.rb

```diff
@@ -43,14 +47,18 @@ def page_title
         @page_title ||= format_string(page["title"]) || site_title
       end
 
+      def site_tagline_or_description
+        site_tagline || site_description
+      end
+
       # Page title with site title or description appended
       # rubocop:disable Metrics/CyclomaticComplexity
       def title
         @title ||= begin
           if site_title && page_title != site_title
             page_title + TITLE_SEPARATOR + site_title
           elsif site_description && site_title
-            site_title + TITLE_SEPARATOR + site_description
+            site_title + TITLE_SEPARATOR + site_tagline_or_description
```

Comment:
If this is the only place `site_tagline_or_description` is used, I’d suggest inlining the method. Feel free to use parentheses.

Reply:
I tried to do that, but it created a style offence: 
* `Metrics/AbcSize: Assignment Branch Condition size for title is too high`
* `Metrics/PerceivedComplexity: Perceived complexity for title is too high`